sql-create-insert-delete/sql-create-insert-delete_solution.sql

```diff
@@ -0,0 +1,47 @@
+USE dsstudent;
+
+#1 Create table
+CREATE TABLE person_newton
+	(person_id SMALLINT, first_name VARCHAR(30), last_name VARCHAR(30), city VARCHAR(30),
+	CONSTRAINT pk_person_newton PRIMARY KEY (person_id));
+	
+#2 Insert a row of data
+INSERT INTO person_newton (person_id, first_name, last_name, city)
+VALUES (1, 'Newton', 'Tran', 'Newport Beach');
+
+#3 Insert two rows of data
+INSERT INTO person_newton (person_id, first_name, last_name, city)
+VALUES (2, 'Truc', 'Tran', 'Newport Beach');
+
+INSERT INTO person_newton (person_id, first_name, last_name, city)

```

Comment:
You can also just comma separate the values to be added after the first "VALUES"